packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -307,11 +305,9 @@ A custom email address allows you to present a branded experience to your end us
 
 Okta sends your super admins a confirmation email after your custom email address is configured and operating correctly. To ensure continuous operation, Okta polls your custom email domain once every 24 hours. If a problem occurs, Okta alerts super admins by email, and Okta-generated emails are sent from the default address `noreply@okta.com` until the problem is resolved.
 
-> **Note:** If you've enabled [Early Access (EA) multibrand customization](https://help.okta.com/okta_help.htm?type=oie&id=csh-branding), your Admin Console navigation is different. See parenthetical notes.
-
 ## Configure a custom email address
 
-1. In the Admin Console, go to **Customizations** > **Emails**. (EA users: go to **Customizations** > **Brands**, and then select the brand you want. In the **Domains** tab, in the **Email** section, click **Edit** then click **Add custom emaildomain**).
+1. In the Admin Console, go to **Customizations** > **Emails**. (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want. In the **Domains** tab, in the **Email** section, click **Edit** then click **Add custom emaildomain**).
```

Comment:
We don't have that Edit button anymore to for `Add custom emaildomain` 

> In the **Domains** tab, in the **Email** section, click "Add Domain" 
Refer my screenshot below - 
![Screen Shot 2023-05-04 at 11 47 32 AM](https://user-images.githubusercontent.com/73907674/236300392-4edbc1f2-7f3e-40a4-a832-942351148e81.png)

Reply:
Fixed. Thanks!